index.html

```diff
@@ -47,38 +55,42 @@ <h4 class="experience-heading">Language</h4>
             <div>
                 <h4 class="experience-heading">Frameworks</h4>
                 <ul class="list-text">
-                    
                     <li><a href="">Bootstrap</a></li>
                     <li><a href="">Ruby on Rails</a></li>
                     <li><a href="">RSPec</a></li>
                     <li><a href="">CapyBara</a></li><li><a href="">Selenium</a></li>
                 </ul>
             </div>
+            <div class="skills-align">
+                <h4 class="experience-heading">Skills</h4>
+                <ul class="list-text">
+                    <li><a href="">Codekit</a></li>
+                    <li><a href="">Github</a></li>
+                    <li><a href="">Codepen</a></li>
+                    <li><a href="">Gitlab</a></li>
+                    <li><a href="">Terminal</a></li>
+                </ul>
+            </div>
         </div>
-        <div class="skills-align">
-            <h4 class="experience-heading">Skills</h4>
-            <ul class="list-text">
-                <li><a href="">Codekit</a></li>
-                <li><a href="">Github</a></li>
-                <li><a href="">Codepen</a></li>
-                <li><a href="">Gitlab</a></li>
-                <li><a href="">Terminal</a></li>
-            </ul>
+        <div class="work-join">
+            <div class="work1">
+                <p class="neutral-color">work</p>
+                <img src="images/Group55.png" alt="">
+            </div>
+            <div class="social-media">
+                <a href="https://angel.co"><img src="images/Angelist.png" alt=""></a>
+                <a href="https://linkedin.com"><img src="images/Linkedin.png" alt=""></a>
+                <a href="https://twitter.com"><img src="images/Twitter.png" alt=""></a>
+                <a href="https://github.com"><img src="images/GitHub.png" alt=""></a>
+                <a href="https://medium.com"><img src="images/Vector.png" alt=""></a>
+            </div>
+            <div class="work2">
+                <img src="images/Group55.png" alt="">
+                <p class="neutral-color">work</p>
+            </div>
```

Comment:
- [ ] Kindly make sure your work arrow pointer is on the left side on mobile view, currently, it's on the right.